### DISCUSSION TITLE
Problem connecting winlogbeat

### DISCUSSION THREAD

**Discussion by canix1** [Security-Onion-Solutions/securityonion#6016] [Unsupported Versions] [answered]
Hi,

I'm trying to install the winlogbeat to send Windows eventlogs to security onion.

Do I need a user name and password in the winlogbeat.yml if so where do I find these?
output.elasticsearch:
  hosts: ["<es_url>"]
  username: "elastic"
  password: "<password>"

When I run the winlogbeat.exe setup I get this;
Exiting: Index management requested but the Elasticsearch output is not configured/enabled

What does this mean?

Thanks

**Comment by dougburks** [comment, marked as answer]
You don't need to authenticate to Elasticsearch or run `winlogbeat.exe setup`. Please follow the instructions at https://docs.securityonion.net/en/2.3/beats.html#winlogbeat.

**Reply by canix1** [reply]
Thanks that solved it!

I'm running SO  2.3.80 -Standard

But now I get :
021-10-28T00:05:07.365+0200    ERROR   [publisher_pipeline_output]     pipeline/output.go:154  Failed to connect to backoff(async(tcp://so2.domain.local:5044)): dial tcp 192.168.1.9:5044: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

Port is opened with so-allow
nmap says it's filtered : 5044/tcp filtered lxi-evntsvc

What could I check?

**Reply by dougburks** [reply]
Are there any network firewalls that could be interfering?
